lib/cisco_node_utils/vxlan_vtep_vni.rb

```diff
@@ -236,5 +236,27 @@ def suppress_arp=(state)
     def default_suppress_arp
       config_get_default('vxlan_vtep_vni', 'suppress_arp')
     end
+
+    def suppress_uuc
+      config_get('vxlan_vtep_vni', 'suppress_uuc', @get_args)
+    end
+
+    def suppress_uuc=(state)
+      if state
+        set_args_keys(state: '')
+        # Host reachability must be enabled for this property
+        VxlanVtep.new(@name).host_reachability = 'evpn'
```

Comment:
@mikewiebe @robert-w-gries I'm a little concerned about having this external dependency call; i.e. the fact that this property is going to flip-flop on each run when the user sets this to a different value using the vxlan_vtep provider. 
I suggest changing this to:

```
      raise RuntimeError, "suppress_uuc dependency: vxlan_vtep host_reachability is not 'evpn'" unless
        VxlanVtep.new(@name).host_reachability == 'evpn'
```

Thoughts?